finagle-redis/src/test/scala/com/twitter/finagle/redis/commands/key/KeyClientIntegrationSuite.scala

```diff
@@ -14,11 +15,13 @@ import org.scalatest.junit.JUnitRunner
 @RunWith(classOf[JUnitRunner])
 final class KeyClientIntegrationSuite extends RedisClientTest {
 
+  val TIMEOUT = 5.seconds
```

Comment:
we use CamelCase for constants, can you change this to Timeout?  an alternative would be to add an `await` method:

```scala
def await[A](a: Future[A]): A = Await.result(a, 5.seconds)
```